ceph-pull-requests/config/definitions/ceph-pull-requests.yml

```diff
@@ -62,3 +62,22 @@
                   - ABORTED
               build-steps:
                 - shell: "sudo reboot"
+      - cobertura:
+          report-file: "src/pybind/mgr/dashboard/frontend/coverage/cobertura-coverage.xml"
+          only-stable: "true"
+          fail-no-reports: "true"
+          fail-unhealthy: "true"
```

Comment:
I don't think you want to fail the builds for unhealthy or unstable, these should just be removed (they both default to `false`)

Reply:
Yup, `fail-unhealthy` can be removed I think